### ISSUE TITLE
More flexibility wrt lyrics files

### ISSUE THREAD

**Issue by diederikdehaas** [MusicPlayerDaemon/ncmpc#90]
In a previous release the lyrics plugin was removed and then I realized how much I liked it.

I also use Cantata as mpd client and that one was still able to retrieve the lyrics. And I found out it stores/caches them in `~/.cache/cantata/lyrics/<artist>/<song-title>.lyrics` on the PC where I'm running it.

Luckily there's a new lyrics 'plugin' and I set `lyrics-autosave = yes` and found out* it stores them in 
`~/.lyrics/<artist> - <song-title>.txt` on my RPi running mpd and that's also where ncmpc is running.

Never wanting to lose the lyrics (files) again, I copied the `.lyrics` files from Cantata's cache to the directory which stores the actual music files, and renamed/moved them so the only difference is in the file extension `.flac` -> `.lyrics`. (in `<music-dir>/<artist>/<album>/<album-sequence-nr>.<song-title-with-underscore-iso-spaces>.flac`; ripped with `abcde`)
Oh yeah, my music files are stored on yet another device in my LAN which shares it via nfs ;-P
While I much prefer storing the lyrics files with the music files, it didn't help with with ncmpc.

*) I found that out by looking at `lyrics/10-hd.sh`
I then also noticed the heuristic is really limited (and `~/.lyrics/` possibly not ideal from XDG POV and not configurable)

So I'd like if the heuristics could be expanded to also check `.lyrics` files and also for files where the only difference is in the file extension, like in my 3rd 'scenario'.

I'm talking here about *retrieving* the lyrics files.
Writing the DL-ed lyrics files in a more flexible way could get much more complicated. It would be cool and convenient, but I'm absolutely fine if I have to do that manually.

**Comment by progman-dt** [comment]
Yes, an option in the config file specifying where to save lyrics would be nice. Something like `lyrics-save-path`. Right now `~/.cache/ncmpc/lyrics` is hardcoded.
